homeassistant/components/switch/__init__.py

```diff
@@ -31,6 +32,7 @@
     discovery.services.BELKIN_WEMO: 'wemo',
     wink.DISCOVER_SWITCHES: 'wink',
     isy994.DISCOVER_SWITCHES: 'isy994',
+    camera.DISCOVER_SWITCHES: 'child_switch',
```

Comment:
Cool, done, I'll check in what I've done tonight in a bit and then tackle the rest over the weekend.